### PR TITLE
feat(web-app): add mdx color palette component

### LINKED ISSUE #569: Add the ColorPalette MDX component from Carbon website
Visual reference
https://www.carbondesignsystem.com/patterns/status-indicator-pattern/#status-palette
https://www.carbondesignsystem.com/patterns/status-indicator-pattern/#status-palette

Code
https://github.com/carbon-design-system/carbon-website/tree/main/src/components/ColorPalette

 MDX

```
<ColorPalette type="categorical" />
<ColorPalette type="grouped" />
<ColorPalette type="sequential" isMono />
<ColorPalette type="sequential" isDiverging />
<ColorPalette twoColumn type="alert" />

<ColorPalette twoColumn type="status" />
<ColorPalette type="status-extended" shouldShowControls={false} />
```

### PR DESCRIPTION
**Pull request by francinelucca**
Closes #569 

#### Testing / reviewing

http://localhost:3000/mdx/color-palette
https://www.carbondesignsystem.com/patterns/status-indicator-pattern/#status-palette

### CODE REVIEW

File: services/web-app/components/color-palette/color-palette.module.scss [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+    #f6f2ff,
+    #e8daff,
+    #d4bbff,
+    #be95ff,
+    #a56eff,
+    #8a3ffc,
+    #6929c4,
+    #491d8b,
+    #31135e,
+    #1c0f30
```
Comment (alisonjoseph) [inline services/web-app/components/color-palette/color-palette.module.scss:277]:
```suggestion
    #f6f2ff,
    #e8daff,
    #d4bbff,
    #be95ff,
    #a56eff,
    #8a3ffc,
    #6929c4,
    #491d8b,
    #31135e,
    #1c0f30
```
```suggestion
    colors.$purple-10,
    colors.$purple-20,
    colors.$purple-30,
    colors.$purple-40,
    colors.$purple-50,
    colors.$purple-60,
    colors.$purple-70,
    colors.$purple-80,
    colors.$purple-90,
    colors.$purple-100
```

Might as well swap these for the variables while we're at it. Same for the other gradients below.

File: services/web-app/components/color-palette/color-palette.module.scss [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+
```
Comment (alisonjoseph) [inline services/web-app/components/color-palette/color-palette.module.scss:12]:
```suggestion
```suggestion
@use '@carbon/react/scss/colors' as colors;
```
```

Comment (alisonjoseph) [inline services/web-app/components/color-palette/color-palette.module.scss:12]:
```suggestion
```suggestion
@use '@carbon/react/scss/colors' as colors;
```
```

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
One small suggestion to import the colors for the css gradients, but otherwise this looks great! 🔥

File: services/web-app/components/color-palette/color-palette.module.scss [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+    #be95ff,
+    #a56eff,
+    #8a3ffc,
+    #6929c4,
+    #491d8b,
+    #31135e,
+    #1c0f30
+  );
+}
+
```
Comment (alisonjoseph) [inline services/web-app/components/color-palette/color-palette.module.scss:280]:
```suggestion
.gradient-purple {
  background: linear-gradient(
    to bottom,
    colors.$purple-10,
    colors.$purple-20,
    colors.$purple-30,
    colors.$purple-40,
    colors.$purple-50,
    colors.$purple-60,
    colors.$purple-70,
    colors.$purple-80,
    colors.$purple-90,
    colors.$purple-100
  );
}

```

**Review by alisonjoseph — DISMISSED** [review]
LGTM!

**Review by andreancardona — DISMISSED** [review]
🔥

**Review by andreancardona — APPROVED** [review]

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**